controllers/fenceagentsremediation_controller_test.go

```diff
@@ -99,6 +99,47 @@ var _ = Describe("FAR Controller", func() {
 		})
 
 		Context("buildFenceAgentParams", func() {
+			Context("build fence agent params", func() {
```

Comment:
this Context has the same description as the above Context, can we have a better description please?

Reply:
Thanks. I will change the comment.